style.css

```diff
@@ -422,34 +201,468 @@ form button:active {
+    line-height: 52px;
+    display: block;
+    margin: 0 auto;
+    margin-bottom: 24px;
+  }
+
+  .about-me p {
+    color: #42526e;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    text-align: center;
+    margin-bottom: 24px;
+  }
+
+  .strength p {
+    color: #344563;
+    font-weight: 600;
+    font-size: 24px;
+    line-height: 32px;
+  }
+
+  .advert p {
+    color: #172b4d;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    text-align: center;
+  }
+
+  .social {
+    padding-bottom: 22px;
+  }
+
+  .social ul {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 0 102px;
+    padding-bottom: 64px;
+  }
+
+  li {
+    list-style: none;
+  }
+
+  .recent-works {
+    width: 375px;
+    padding-top: 22px;
+    padding-bottom: 62px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .recent-works h1 {
+    width: 70%;
+    color: #172b4d;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    text-align: center;
+  }
+
+  .multi-stories {
+    display: grid;
+  }
+
+  .story {
+    width: 327px;
+    height: 474px;
+    margin: 0 auto 24px auto;
+    border-radius: 8px;
+    background: #ebf0ee;
+  }
+
+  .story .content ul {
+    display: flex;
+    justify-content: space-between;
+    margin: 0 11px 29px 11px;
+  }
+
+  .story .content li {
+    background: #ebf0ee;
+    border-radius: 4px;
+    font-weight: 600;
+    font-size: 12px;
+    line-height: 16px;
+    padding: 8px 12px;
+  }
+
+  .btn {
+    display: flex;
+    justify-content: center;
+    margin-bottom: 100px;
+  }
+
+  /* about me */
+  .strength {
+    width: 327px;
+  }
+
+  /* Styles for footer */
+
+  .footer {
+    width: 375px;
+    background-color: #fff;
+    margin: 0 auto;
+    margin-bottom: 10px;
+  }
+
+  form {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    background-image: url("./images/illustration-contact-form.svg");
+    background-repeat: no-repeat;
+    background-position: bottom right;
+  }
+
+  footer h3 {
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    color: #172b4d;
+    text-align: center;
+    margin: 0 24px;
+    margin-bottom: 58px;
+  }
+
+  form input::-webkit-input-placeholder {
+    color: #6b778c;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+  }
+
+  form button {
+    margin: 0 auto;
+  }
+
+  .email {
+    margin-top: 24px;
+  }
+
+  .footer p {
+    color: #42526e;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    text-align: center;
+    margin-bottom: 330px;
+  }
+
+  .bar {
+    width: 150px;
+    border-bottom: 10px solid rgb(36, 35, 35);
+    margin: 0 auto;
+    border-radius: 5px;
+    background-color: #fff;
+  }
+
+  .space {
+    height: 10px;
+  }
+}
+
+/* styles for desktop version */
+
+.desktop {
+  display: none;
+}
+
+@media only screen and (min-width: 376px) {
```

Comment:
Your project is fantastic and everything is working as intended except our breakpoint needs a bit of adjustment.
- [ ] I suggest using 768px or wider for breakpoint that makes us 100% cover all mobile screens before breaking into desktop/tablet for currently our min-width is set to `376px` which makes us cover a large number of mobiles but for example, iPhone 6/7/8 plus are `414px` wide so they will see the desktop view.